Gemfile

```diff
@@ -75,7 +75,7 @@ group :jekyll_optional_dependencies do
   gem "jekyll-redirect-from"
   gem "kramdown", "~> 1.14"
   gem "mime-types", "~> 3.0"
-  gem "rdoc", "~> 5.0"
+  gem "rdoc", "~> 6.0"
```

Comment:
Maybe still allow `5.x` for Ruby 2.1?

```
gem "rdoc", [">= 5.0", "< 7.0"]

Reply:
like we have for other gems in similar situations.
```ruby
gem "rdoc", RUBY_VERSION >= "2.2.2" ? "~> 6.0" : "~> 5.1"
```